src/logic/map_objects/tribes/productionsite.cc

```diff
@@ -1020,8 +1018,8 @@ void ProductionSite::train_workers(Game& game) {
 }
 
 void ProductionSite::notify_player(Game& game, uint8_t minutes, FailNotificationType type) {
-	if (last_stat_percent_ == 0 ||
-	    (last_stat_percent_ <= descr().out_of_resource_productivity_threshold() &&
+	if (last_stat_percent_ / 10 == 0 ||
+	    (last_stat_percent_ / 10 <= descr().out_of_resource_productivity_threshold() &&
```

Comment:
What is this '/10' ?

Reply:
It's better to use `get_statistics_percent()` and lose the `/ 10`, because I asked myself that question too.

Reply:
I used get_actual_statistics now, because I believe get_statistics_percent might be superfluous now. But I have to check whether it is used somewhere else.